src/gui/widgets/GrooveView.cpp

```diff
@@ -26,37 +26,35 @@
 GrooveView::GrooveView( ) :
 	QWidget()
 {
-	setMinimumWidth( 250 );
-	setMinimumHeight( 210 );
-	setMaximumWidth( 250 );
-	resize( 250, 220 );
-
 	setWindowIcon( embed::getIconPixmap( "note_double_whole" ) );
 	setWindowTitle( tr( "Groove" ) );
 
+	m_layout = new QVBoxLayout();
+	this->setLayout( m_layout );
+
 	m_dropDown = new QComboBox(this);
-	// insert reverse order
-	m_dropDown->insertItem(0, "Half Swing" , QVariant::fromValue(5) );
-	m_dropDown->insertItem(0, "Experiment" , QVariant::fromValue(4) );
-	m_dropDown->insertItem(0, "Midi Swing" , QVariant::fromValue(3) );
-	m_dropDown->insertItem(0, "Hydrogen Swing" , QVariant::fromValue(2) );
-	m_dropDown->insertItem(0, "Straight" , QVariant::fromValue(1) );
+	// Insert reverse order.
+	m_dropDown->insertItem(0, tr("Experiment swing") , QVariant::fromValue(5) );
+	m_dropDown->insertItem(0, tr("Hydrogen swing") , QVariant::fromValue(4) );
+	m_dropDown->insertItem(0, tr("Half swing") , QVariant::fromValue(3) );
+	m_dropDown->insertItem(0, tr("MIDI swing") , QVariant::fromValue(2) );
+	m_dropDown->insertItem(0, tr("No swing") , QVariant::fromValue(1) );
```

Comment:
"Straight" is well recognized, no? Also, depending on where this menu appears/what it looks like, perhaps the redundant "swing" could be removed.

Reply:
I'm not aware of "Straight"'s recognition, but I went with "No swing" for consistency with the `StudioController` where there's "No controller". And because of "No swing", the other options have "swing". The dropdown menu appears in the window of `Groove`, first image in the first comment.